tests/flow_control.rs

```diff
@@ -345,49 +345,55 @@ fn stream_close_by_data_frame_releases_capacity() {
         // Send request
         let mut s1 = client.send_request(request, false).unwrap();
 
-        // This effectively reserves the entire connection window
-        s1.reserve_capacity(window_size);
+            // This effectively reserves the entire connection window
+            s1.reserve_capacity(window_size);
```

Comment:
🤔

Reply:
Oh, looks like this happened while rebasing, my bad - I'll fix that.